zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -104,30 +104,32 @@ lazy_static! {
     /// as described in [protocol specification §7.10.1][7.10.1].
     ///
     /// [7.10.1]: https://zips.z.cash/protocol/protocol.pdf#zip214fundingstreams
-    pub static ref FUNDING_STREAM_HEIGHT_RANGES: HashMap<Network, std::ops::Range<Height>> = {
+    // TODO: Move the value here to a field on `NetworkParameters` (#8367)
+    pub static ref FUNDING_STREAM_HEIGHT_RANGES: HashMap<String, std::ops::Range<Height>> = {
```

Comment:
It should be fine to use the testnet values on regtest, double check chainparams in zcashd, doesn't matter either way except for interoperability or compatibility between zcashd and zebrad.

Reply:
In Zcashd regtest [src/chainparams.cpp:751], there is differing logic for loading funding streams. It is configurable in [src/init.cpp:1328].

Reply:
Updated the TODO to reflect that in https://github.com/ZcashFoundation/zebra/pull/8368/commits/8b7455990c338e017c4278d3ea2cfe68a39471ca.

I left "if required" because there is _some_ (very minimal) checkpoint data in zcashd (even for Regtest), so I'm not yet sure if we can get rid of the checkpoints altogether here.